extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalFooter_index.php

```diff
@@ -1,4 +1,4 @@
-<footer class="wds-global-footer <?= isset( $model['international_header'] ) ? 'wds-is-international' : 'wds-is-en' ?>">
+<footer class="wds-global-footer<?= isset ( $model['header'] ) ? '' : ( isset( $model['international_header'] ) ? ' wds-is-international' : ' wds-is-en' ); ?>">
```

Comment:
Nested ternary operators in the template?
Can we convert it to to regulat if block to increase readability?